onfido/helpers.py

```diff
@@ -50,23 +47,15 @@ def create_check(
     Returns a new Check object, and creates the child Report objects.
 
     """
-    if check_type != "standard":
-        raise ValueError(
-            f"Invalid check_type '{check_type}', currently only 'standard' "
-            "checks are supported."
-        )
-    if not isinstance(reports, (list, tuple)):
-        raise ValueError(
-            f"Invalid reports arg '{reports}', must be a list or tuple " "if supplied."
-        )
     data = {
-        "type": check_type,
-        "reports": [{"name": r} for r in reports],
+        "applicant_id": applicant.onfido_id,
+        "report_names": report_names,
     }
     # merge in the additional kwargs
     data.update(kwargs)
-    response = post("applicants/{}/checks".format(applicant.onfido_id), data)
+    response = post("checks", data)
     check = Check.objects.create_check(applicant=applicant, raw=response)
-    for report in response["reports"]:
+    reports = get(f"reports?check_id={check.onfido_id}")
+    for report in reports:
```

Comment:
The check API now returns report ids, not the full report, so we have to make a `GET` to fetch those. I think it may be possible to get the reports using `?expand=reports`, but I cannot confirm this and the docs are ambiguous.